docs/usage/gitlab-bot-security.md

```diff
@@ -83,6 +83,11 @@ Please see the [renovate-bot/renovate-runner README on GitLab](https://gitlab.co
 We're trying to find a workable design for the GitLab app, so we can enable it safely again.
 If you have any ideas, open a [discussion](https://github.com/renovatebot/renovate/discussions) and let us know!
 
+## Current status
+
+**Update:** GitLab now introduced Group Access Tokens & API for paid & self-hosted instances, but a good permission setup/flow is still not possible.
+Check out the [relevant issue](https://gitlab.com/gitlab-org/gitlab/-/issues/346298) we created.
```

Comment:
```suggestion
Check out [GitLab issue #346298](https://gitlab.com/gitlab-org/gitlab/-/issues/346298).
```

Can we do something like this? Link texts should be clear about where you'll end up. 😉